docs/pages/_app.js

```diff
@@ -77,6 +56,7 @@ ponyfillGlobal.muiDocConfig = {
       '@mui/x-date-pickers': getMuiPackageVersion('x-date-pickers', muiCommitRef),
       '@mui/x-date-pickers-pro': getMuiPackageVersion('x-date-pickers-pro', muiCommitRef),
       'date-fns': 'latest',
```

Comment:
```suggestion
```
Could we look into the feasibility of dropping it from here as well? 🤔 
It's probably a leftover after moving all the pickers docs to use `dayjs` by default instead of `date-fns`.
But we'd still need to double-check that the cases, where `AdapterDateFns` is used in demos that this library is included correctly. 🤔 

cc @flaviendelangle

Reply:
I think we can remove it.
It's only used to specify the dependency version, but `date-fns` is not added here anymore